Embedded Metadata.js

```diff
@@ -913,11 +954,14 @@ function finalDataCleanup(doc, newItem) {
 			newItem.tags = tags;
 		}
 	}
+	if (newItem.tags.length > 0) newItem.tags = scrubLowercaseTags(newItem.tags);
+	
+	/* what else is automatically adding tags? if it's bad data, don't import it, but resetting all tags (per below) kills good LD-JSON tags
 	else {
 		// Unless called from another translator, don't include automatic tags,
```

Comment:
What else is automatically adding tags? Resetting all tags here—the default—kills (now) good LD-JSON tags. Better to just avoid anything that adds garbage tags if we want to be vigilant, rather than adding and immediately wiping it out?